pkg/models/nf_address_list.go

```diff
@@ -4,8 +4,31 @@
 
 package models
 
+import "openpitrix.io/notification/pkg/util/idutil"
+
 type NFAddressList struct {
 	NFAddressListId string `gorm:"column:nf_address_list_id"`
 	NotificationId  string `gorm:"column:notification_id"`
 	AddressListId   string `gorm:"column:address_list_id"`
 }
+
+const (
+	NFAddressListIdPrefix = "x-"
```

Comment:
Change to meaningful prefix, such as nfa-

Reply:
done